CHANGELOG.md

```diff
@@ -15,6 +15,8 @@
 - Updated translations: `om-ET`, `pa-IN`, `kn-IN`, `so-ET`, `ti-ET`, `mr-IN`
 - Show search results when overdue search results are loaded
 - Handle user interactions for overdue search results
+- Show search results when overdue search results are loaded
```

Comment:
Seems like we added both the change log entries in this PR itself. Can you change that?

Reply:
Good catch, fixing it 👍🏾